app/adapters/bookmarklet.js

```diff
@@ -1,27 +1,28 @@
 import BasicAdapter from "./basic";
+import Ember from 'ember';
+const { on } = Ember;
 
 export default BasicAdapter.extend({
   name: 'bookmarklet',
 
   inspectedWindow: function() {
     return window.opener || window.parent;
-  }.property(),
+  },
 
   inspectedWindowURL: function() {
     return loadPageVar('inspectedWindowURL');
-  }.property(),
+  },
 
   sendMessage: function(options) {
     options = options || {};
     this.get('inspectedWindow').postMessage(options, this.get('inspectedWindowURL'));
   },
 
-  _connect: function() {
-    let self = this;
+  _connect: on('init', () => {
```

Comment:
this needs to be `on('init', function() {` otherwise `this` wouldn't refer to the object instance. Use `function` instead of `=>` for `Ember.on`, `Ember.observer`, `Ember.computed` so you that `this` would be the object itself.